### PR TITLE
[no ticket][risk=no] dataset query efficiency

### PR DESCRIPTION
**Pull request by freemabd**
Make dataset queries faster

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java [hunk 1]
```diff
@@ -76,7 +76,7 @@
       "is_standard = %s and concept_id in (select distinct c.concept_id\n"
           + "from `${projectId}.${dataSetId}.cb_criteria` c\n"
           + "join (${childLookup}) a\n"
-          + "on (c.path like concat('%%.', a.id, '.%%') or c.path like concat('%%.', a.id) or c.path like concat(a.id, '.%%') or c.path = a.id)\n"
+          + "on (c.path like concat('%%.', a.id, '.%%') or c.path like concat('%%.', a.id))\n"
```
Comment (jscherdin) [inline api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java:79]: All of your SQL ends with \n. I would probably replace that with a "space"

**Review by jscherdin — CHANGES_REQUESTED** [review]
Looks good besides my one comment

File: api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java (hunk 1 above)
Comment (freemabd) [reply api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java:79]: space won't do a carriage return.

**Review by jscherdin — APPROVED** [review]
LGTM

**Pull request merged**